packages/ibm-products/src/components/Datagrid/Datagrid/DatagridContent.tsx

```diff
@@ -237,7 +236,6 @@ export const DatagridContent = ({
           [`${blockClass}__with-pagination`]: DatagridPagination,
           [`${blockClass}__dense-header`]: useDenseHeader,
         })}
-        title={gridTitle}
```

Comment:
This prop isn't a `title` attribute but an actual prop for the `TableContainer` from `@carbon/react` which displays an optional title for the table.
![Screenshot 2024-06-24 at 3 46 00 PM](https://github.com/carbon-design-system/ibm-products/assets/10215203/e9d1fbf6-c3a3-43a5-b7dd-def700905ae3)

Reply:
oooh i see. ok. my mistake. will revert those changes.